src/components/UserInfo.js

```diff
@@ -2,17 +2,29 @@ export default class UserInfo {
   constructor() {
     this._profileName = document.querySelector(".profile__title");
     this._profileJob = document.querySelector(".profile__description");
+    this._profileAvatar = document.querySelector(".profile__image");
```

Comment:
You need to pass 3 selectors into `UserInfo` and use them for seraching the elements according to the task